src/zjs_event.c

```diff
@@ -443,17 +443,6 @@ void *zjs_event_get_user_handle(jerry_value_t obj)
     return NULL;
 }
 
-static void zjs_event_cleanup(void *native)
-{
-    jerry_release_value(zjs_event_emitter_prototype);
```

Comment:
The prototype never gets freed now - so as discussed in IRC you should acquire prototype in make_emitter and release it once each time an emitter is freed. Be careful that when you *create* the prototype, which adds one reference, that first emitter doesn't end up with two references or it will never get freed. That will be a little ugly.

Reply:
The latest commit contains code to free the prototype, give it a look.